src/main/java/frc/robot/commands/MoveToNextBar.java

```diff
@@ -41,14 +42,14 @@ public MoveToNextBar() {
                 ).withTimeout(DROP_PLACERS_TIMEOUT.get()),
                 new MoveGenericSubsystem(climberWinch, ClimberWinch.UP_SPEED),
                 new InstantCommand(() -> {
-                    leftPlacer.setNeutralMode(NeutralMode.Brake);
-                    rightPlacer.setNeutralMode(NeutralMode.Brake);
+                    leftPlacer.setIdleMode(IdleMode.kBrake);
+                    rightPlacer.setIdleMode(IdleMode.kBrake);
                 }),
                 new MoveBothPlacersToNextBar(),
                 new MoveGenericSubsystem(climberWinch, ClimberWinch.DOWN_SPEED).withTimeout(HOOK_BAR_TIMEOUT.get()),
                 new InstantCommand(() -> {
-                    leftPlacer.setNeutralMode(NeutralMode.Coast);
-                    rightPlacer.setNeutralMode(NeutralMode.Coast);
+                    leftPlacer.setIdleMode(IdleMode.kCoast);
```

Comment:
why would you want to change it back to coast?

Reply:
That's what we've been told by mechanics, maybe this has changed since